src/routes/library.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { H1 } from '$lib/shared/ui';
+  import { likesStore } from '$lib/features/like-episode';
+  import { EpisodeCard } from '$lib/widgets/episode';
+
+  $: episodes = Array.from($likesStore.values());
+</script>
+
+<!-- Sign in card -->
+
+<!-- Favorites -->
+<section class="flex flex-col p-4 gap-4 items-stretch">
+  <H1>Favorites</H1>
+  {#each episodes as episode (episode.id)}
+    <EpisodeCard {episode} playing={false} />
+  {/each}
+</section>
+
+<!-- Offline -->
```

Comment:
issue: let's think of `routes` as being on the app level, ideally keeping as little page and composition logic here as possible. 

suggestion: I would suggest moving this code to `lib/pages/library/ui/library-page.svelte`